src/app.js

```diff
@@ -25,11 +41,77 @@ const quoteData = [
   },
 ];
 
-$(document).ready(function() {
+
+const quoteList = new QuoteList(quoteData);
+const orderList = new OrderList();
+
+
+let quoteTemplate;
+let tradeTemplate;
+let orderTemplate;
+
+$(document).ready( () => {
+
+  let bus = {}; //can trigger/subscribe to BB events
+  bus = _.extend(bus, Backbone.Events);
+
+
+  quoteTemplate = _.template($('#quote-template').html());
+  tradeTemplate = _.template($('#trade-template').html());
+  orderTemplate = _.template($('#order-template').html());
+
   const quotes = new QuoteList(quoteData);
+  // const orders = new OrderList();
+
   const simulator = new Simulator({
     quotes: quotes,
   });
 
-  simulator.start();
+    simulator.start();
+
+  const order = new Order ({
+    bus: bus,
+  });
+
+// I'm making an instance of quoteListView and I'm saying that el is the quotes-container, the model to use is quoteList, which I've filled with some data, the template to use is the quote Template and the bus to use is bus (that was set up at top of document ready. )
+
+
+  const quoteListView = new QuoteListView({
+    el: '#quotes-container',
+    model: quoteList,
```

Comment:
By passing `quoteList` here we are creating a view for a collection of quotes that is not connected to the simulator at all (and thus there appears to be no automatic change in prices).

If we insted write `model: quotes,` for this line then we'll be connecting this `QuoteListView` to the same collection of quotes as the simulator receives, which should solve the issue.